sources/packages/backend/libs/utilities/src/integration-utils.ts

```diff
@@ -22,3 +23,20 @@ export function getTotalYearsOfStudy(completionYears: string): number {
       return 1;
   }
 }
+
+/**
+ * Replace the line breaks in the given text.
+ * @param data data to replace the line break.
+ * @param options replace options:
+ * - `replaceText`: text to replace the line break.
+ * @returns line break replaced string.
+ */
+export function replaceLineBreaks(
```

Comment:
Maybe part of the PR or not but these methods are nice ones to have small unit tests added.